databuilder/databuilder/extractor/oracle_metadata_extractor.py

```diff
@@ -0,0 +1,48 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import (  # noqa: F401
+    Any, Dict, Iterator, Union,
+)
+
+from pyhocon import ConfigFactory, ConfigTree  # noqa: F401
+
+from databuilder.extractor.base_oracle_metadata_extractor import BaseOracleMetadataExtractor
+
+
+class OracleMetadataExtractor(BaseOracleMetadataExtractor):
+    """
+    Extracts Oracle table and column metadata from underlying meta store database using SQLAlchemyExtractor
+    """
+
+    def get_sql_statement(self, use_catalog_as_cluster_name: bool, where_clause_suffix: str) -> str:
+        if use_catalog_as_cluster_name:
```

Comment:
This feels unnecessary. Why would we use table name as cluster name? I'd keep it more simple and just have default cluster name + get rid of this book completely.

Reply:
You're right. I've ceated a default cluster name to Oracle extractor in line 51 (file base_oracle_metadata_extractor.py).